raiden/api/webui/static/endpointConfig.js

```diff
@@ -1,5 +1,5 @@
-const backendUrl='http://localhost:5000'; 
-const nodeAddress = '0x720DD4A77bB906b5a7FC847a6f758ab20b84f214'; 
+const backendUrl='http://localhost:5004'; 
+const nodeAddress = '0x0C46357428Dd9b4A1f951Df8B4868b0D972644AD'; 
 const rnsDomain = null 
 const chainEndpoint = 'http://localhost:4444'; 
```

Comment:
watch out here

Reply:
that file is really annoying, but i think now it's just spacing, we should add that to the .gitignore file i think

Reply:
afaic you can just add this to the .gitignore file in this PR :man_shrugging:

Reply:
it was already added but the rule was wrong, now it should work but it will start working after this PR